src/pages/example--hello-world.mdx

```diff
@@ -382,23 +404,27 @@ async function unlock(ref, { from, using }): Promise<TxHash> {
     .attachSpendingValidator(from)
     .complete();
 
-  const signedTx = await tx
-    .sign()
-    .complete();
+  const signedTx = await tx.sign().complete();
 
   return signedTx.submit();
 }
 ```
 
-<Callout>
-  As you imagine, we can run this script with the following incantation:
+Run this script with the transaction hash obtained from the previous command locking the funds:
 
 ```
-deno run --allow-net --allow-read hello_world-unlock.ts
+deno run --allow-net --allow-read --allow-env hello_world-unlock.ts 8559f57234407204d8e9a6bf57ef6943c65ec7119eb1c2ca6224f8bad8e71c1e
 ```
 
-</Callout>
```

Comment:
Why are we getting rid of all of the CallOut components? I'm quite a fan of how they look.

Reply:
I removed the two larger callouts as they are too big for what they are designed for ([reference](https://nextra.site/docs/docs-theme/built-ins/callout#example)).

Reply:
I disagree @jacfra, they worked perfectly fine in this context.